config/manifest/eksa-components.yaml

```diff
@@ -3199,6 +3199,20 @@ spec:
           status:
             description: CloudStackDatacenterConfigStatus defines the observed state
               of CloudStackDatacenterConfig
+            properties:
```

Comment:
This part was generated when I ran `make release-manifests`, it was probably something Cloudstack should've added but missed.